ivadomed/inference.py

```diff
@@ -110,10 +111,11 @@ def get_onehotencoder(context: dict, folder_model: str, options: dict, ds: Datas
     metadata_dict = joblib.load(Path(folder_model, 'metadata_dict.joblib'))
     for idx in ds.indexes:
         for i in range(len(idx)):
-            idx[i]['input_metadata'][0][context['FiLMedUnet']['metadata']] = options['metadata']
+            idx[i]['input_metadata'][0][context[ConfigKW.FILMED_UNET][ModelParamsKW.METADATA]] = options['metadata']
             idx[i]['input_metadata'][0]['metadata_dict'] = metadata_dict
 
-    ds = imed_film.normalize_metadata(ds, None, context["debugging"], context['FiLMedUnet']['metadata'])
+    ds = imed_film.normalize_metadata(
+        ds, None, context[ConfigKW.DEBUGGING], context[ConfigKW.FILMED_UNET][ModelParamsKW.METADATA])
```

Comment:
Might want to eventually add a check for `ConfigKW.FILMED_UNET` field first before invoking its keys.

Reply:
Done